conf/app.ini

```diff
@@ -74,6 +74,11 @@ ORG_PAGING_NUM = 50
 ; Number of repos that are showed in one page
 REPO_PAGING_NUM = 15
 
+[ui.meta]
+AUTHOR = Gitea - Git with a cup of tea
+DESCRIPTION = Gitea: Git with a cup of tea
```

Comment:
For some reason I was thinking it was same as APP_NAME. Corrected in new commit.